docs/user-guide/unaligned-data.ipynb

```diff
@@ -189,7 +189,11 @@
    "metadata": {},
    "source": [
     "This is essentially the same figure as the scatter plot for the original `data`.\n",
-    "The only difference is the \"grid\" (the bin edges) that is stored alongside the data.\n",
+    "The differences are:\n",
```

Comment:
Updates here look ok. We have no updated notebooks making use of `sc.filter` i.e new filtering functionality. Is that coming later as part of further docs updates.

Reply:
There is no notebook, but it should show up based on the docstring here: https://scipp.github.io/generated/scipp.filter.html#scipp.filter (as a new overload)

Reply:
Maybe a better answer: I will add notebooks in the next step, when we can actually support filtering by actual metadata rather than just pulse-times or TOF.

Reply:
@SimonHeybrock yes that makes total sense. When hooked to to `attributes` or equivalent maybe with some simple thresholds.